main_routers/memory_router.py

```
@@ -0,0 +1,230 @@
+        # 1. 重命名记忆文件
+        old_filename = f'recent_{old_name}.json'
+        new_filename = f'recent_{new_name}.json'
+        old_file_path = str(cm.memory_dir / old_filename)
+        new_file_path = str(cm.memory_dir / new_filename)
+        
+        # 检查旧文件是否存在
+        if not os.path.exists(old_file_path):
+            logger.warning(f"记忆文件不存在: {old_file_path}")
+            return JSONResponse({"success": False, "error": f"记忆文件不存在: {old_filename}"}, status_code=404)
+        
+        # 如果新文件已存在，先删除
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)